### PR TITLE
Fixes #36884 - Make more strings translatable and extract strings for Katello 4.11

### PR DESCRIPTION
**Pull request by ianballou**
Besides extracting strings, this PR makes (what should be) the remaining unextractable strings extractable.  Angular breadcrumb labels use the translate filter to become translated, but the gettext grunt extraction task cannot pick them up for shipment to transifex. That means that, if translations are already existing in the locale files, the breadcrumbs get translated, but if not, they go completely untranslated.

To work around this, I've added variables containing the needed strings in related controllers with some comments to help future maintainers. This is a short term workaround for the remaining time that Angular has in our ecosystem.

To test, just poke around the UI and make sure Angular breadcrumbs still look fine. They shouldn't be affected unless I messed up a translation import somewhere.

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/activation-keys.controller.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ angular.module('Bastion.activation-keys').controller('ActivationKeysController',
     ['$scope', '$location', 'translate', 'Nutupane', 'ActivationKey', 'CurrentOrganization', 'experimentalLabsSetting',
     function ($scope, $location, translate, Nutupane, ActivationKey, CurrentOrganization, experimentalLabsSetting) {
 
+        // Labels so breadcrumb strings can be translated
```
Comment (chris1984) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/activation-keys.controller.js:22]: Thanks for leaving comments :)

**Review by chris1984 — APPROVED** [review]
Looks good, send it.

**Comment by ianballou** [comment]
Just need to fix the tests

**Comment by ianballou** [comment]
@chris1984 my latest push should fix all the broken tests.

**Review by chris1984 — APPROVED** [review]
Code looks fine, thanks for taking the time to do this. Went through and checked a few pages and the breadcrumbs look fine. ACK

Thanks for putting comments in the JS Controllers too

**Pull request merged**